Mathlib/Topology/Algebra/Constructions/DomMulAct.lean

```diff
@@ -0,0 +1,112 @@
+/-
+Copyright (c) 2024 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import Mathlib.Topology.Homeomorph
+import Mathlib.GroupTheory.GroupAction.DomAct.Basic
+
+/-!
+# Topological structure on `DomMulAct _`
+
+In this file we define topology on `DomMulAct _` and prove basic facts about this topology.
```

Comment:
Could you explain here that the topology on `DomMulAct X` is the same as the one on `X`?